app/lib/getNearbyServices.js

```diff
@@ -1,23 +1,43 @@
-const log = require('../../app/lib/logger');
+const log = require('../lib/logger');
 const esClient = require('./esClient');
-const filterServices = require('./filterServices');
+const addMessages = require('./addMessages');
+const getDateTime = require('./getDateTime');
 const VError = require('verror').VError;
 const esGetPharmacyHistogram = require('./promHistograms').esGetPharmacy;
+const addMessagesHistogram = require('../lib/promHistograms').addMessages;
 
 async function getNearbyServices(searchCoordinates, limits) {
   let endTimer;
-  let pharmacies;
+  let nearbyPharmacies;
+  let openPharmacies;
 
   try {
     endTimer = esGetPharmacyHistogram.startTimer();
-    pharmacies = await esClient.getPharmacies(searchCoordinates, limits.searchRadius);
+    openPharmacies = await esClient.getOpenPharmacies(
```

Comment:
I'd like to see the timings for each individual query as well as the overall query execution time. I think it would be useful to keep the overall time as when/if these 2 queries are combined into one the same metric can be used to compare. However, for the time there are 2 queries I'd like to know what the difference in execution times are.